lib/packaging/git_helpers.rb

```diff
@@ -49,15 +49,16 @@ def tag_exists?(name = nil)
       git("tag").include?(name)
     end
 
-    # Read the package version from the first .spec file in the packages/
-    # subdirectory.
+    # Read the package version from specfile_name. If specfile_name is 'nil',
+    # use the first .spec file in the packages/ subdirectory.
     #
     # @return String
     #
-    def spec_version
+    def spec_version(specfile_name = nil)
+      specfile_name |= Dir.glob("package/*.spec").first 
```

Comment:
Ah, this was already present in the original code but should be improved anyway:

The [default directory](https://github.com/openSUSE/packaging_rake_tasks/blob/master/lib/packaging/configuration.rb#L73) for creating the package is `package/`, but it is [possible to customize it](https://github.com/openSUSE/packaging_rake_tasks/blob/master/lib/packaging/configuration.rb#L40-L41) in the application's `Rakefile`. So the correct solution is to use `Packaging::Configuration.instance.package_dir` here:

```suggestion
      specfile_name |= Dir.glob("#{Packaging::Configuration.instance.package_dir}/*.spec").first 
```